bridge-token-factory/src/lib.rs

```diff
@@ -362,11 +324,12 @@ impl BridgeTokenFactory {
         assert_self();
         assert!(verification_success, "Failed to verify the proof");
         self.record_proof(&proof);
-        ext_nep21::transfer(
+        ext_fungible_token::ft_transfer(
             recipient,
             amount.into(),
+            None,
             &token,
-            env::attached_deposit(),
+            NO_DEPOSIT,
```

Comment:
Shouldn't we attach here 1 yoctoNEAR deposit according to the rules of `ft_transfer`?

Reply:
This is correct. Updated it.